### ISSUE TITLE
Add tags for releases

### ISSUE THREAD

**Issue by nirvdrum** [alehander42/matchete#1]
machete has had several releases on RubyGems, but none of them seem to be tagged in the repository.  It'd be helpful to check out the source for each released version.

**Comment by alehander92** [comment]
You're right, however I start tagging them from the current release(I am afraid that tagging the previous releases would require rewriting history? or unnecessary merges)

**Comment by nirvdrum** [comment]
Tagging shouldn't require rewriting history.  They're basically named pointers to existing commits (unless you do an annotated tag, in which case it because a commit itself).  You even need to specially push the (`git push --tags`).  More details can generally be found here:

http://git-scm.com/book/en/v2/Git-Basics-Tagging

The only thing is it might be a bit annoying since you'd need to track down the exact SHA used for each gem release.
